.github/workflows/code-styling.yml

```diff
@@ -45,25 +57,25 @@ jobs:
             styler::style_file()
         shell: Rscript --no-init-file {0}
 
-
       ## Second, style the python code ##
-      - name: Format python files
+      - name: Style Python code for each analysis module
         uses: chartboost/ruff-action@v1
         with:
-          args: format analyses/
+          # TODO: This is grabbing everything under analyses, with and without quotes
+          args: format analyses/${{ matrix.module }}/
 
```

Comment:
Ah! I remember seeing this but didn't try it, since I went to look for more docs on whether `src:` would do what I thought...and then promptly didn't follow up...